packages/lisk-p2p/src/peer.ts

```diff
@@ -13,45 +13,65 @@
  *
  */
 /* tslint:disable:interface-name */
+import { Logger, PeerState } from './p2p_types';
+
+// TODO: Use to create outbound socket connection inside peer object.
+// TODO: const socketClusterClient = require('socketcluster-client');
 
 export interface IPeerConfig {
 	readonly clock?: Date;
 	readonly height: number;
 	readonly httpPort?: number;
```

Comment:
`httpPort` probably not needed?

Reply:
We may want to allow nodes to report their HTTP ports to each other but I guess this should be part of the nodeStatus object (along with the height).

The nodeStatus is meant to be the alternative to peer headers so the API module could tell us what the httpPort should be to include as part of the nodeStatus; so we can report it to other peers that way.

Reply:
should we discuss these fields and how it's going to be used?